packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -581,6 +688,26 @@ class _TableState<T> extends State<_Table<T>> {
     scrollController = ScrollController();
   }
 
+  @override
+  void didUpdateWidget(_Table oldWidget) {
+    super.didUpdateWidget(oldWidget);
+
+    cancel();
+
+    // Detect selection changes but only care about scrollIntoView.
+    addAutoDisposeListener(oldWidget.selectionNotifier, () {
+      setState(() {
+        final selection = oldWidget.selectionNotifier.value;
+        if (selection.scrollIntoView) {
+          final newPosition = selection.displayRow * defaultRowHeight;
```

Comment:
can you re-use the code in _moveSelection?

Reply:
I have a todo to support animation and I've added to factor that out of _moveSelection.